api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -157,8 +157,16 @@ public void exportWorkspaces(List<Long> workspaceIds) {
   private RdrResearcher toRdrResearcher(DbUser dbUser) {
     RdrResearcher researcher = new RdrResearcher();
     researcher.setUserId((int) dbUser.getUserId());
-    researcher.setCreationTime(dbUser.getCreationTime().toLocalDateTime().atOffset(offset));
-    researcher.setModifiedTime(dbUser.getLastModifiedTime().toLocalDateTime().atOffset(offset));
+    if (null != researcher.getCreationTime()) {
+      researcher.setCreationTime(dbUser.getCreationTime().toLocalDateTime().atOffset(offset));
+    } else {
+      researcher.setCreationTime(OffsetDateTime.now());
```

Comment:
It's super surprising to see this conversion code take a null value from the DB and set it to the current timestamp in the RDR export call. I'm still kind of scratching my head about this... what's the motivation behind this choice? Is the RDR API unwilling to accept null values? Are they okay with receiving data about researchers whose creation time is constantly changing?

This absolutely needs some comment in the code explaining why we are making a somewhat-surprising choice.

Reply:
I will add comments about this, right now i wanted to solve the creation_date issue with liquibase just so it unblocks the test build.